src/command-configs/renderHelpPage.pug

```diff
@@ -44,61 +54,70 @@ html
-                              th.center
-                                | [#{arg.label}]
-                        tr
-                          td #{commandName}
-                          td
-                            | &nbsp;$&nbsp;
-                          each arg, argKey in preset.args
-                            td.center
-                              if arg.type_one_of
-                                each one in arg.type_one_of
-                                  div.mb-sm #{one}
-                              if arg.type_rule
-                                div #{arg.type_rule}
+                each preset, presetId in cfg.command.presets
+                  div(class="preset " + preset.repos.join(' '))
+                    h6.mb-md(id="link-"+presetId) #{commandStart} #{ commandName } #{presetId === 'default' ? '' : presetId}
+                    p.mb-md.muted Works in these repos: [#{preset.repos.join(', ')}]
+                    -let defaultArgs = []
+                    if preset.args && Object.values(preset.args).length > 0
+                      div.preset-args.mb-lg.ml-md
+                        each arg, argKey in preset.args
+                          div.row.mb-sm
+                            // output only arrays with more than 1 element or regex (type_rule)
+                            if arg.type_one_of && arg.type_one_of.length > 1
+                              div.col-4.lowercase
+                                | --#{arg.label}
+                              div.col-8.lowercase
+                                each one, i in arg.type_one_of
+                                  span.mb-sm
+                                    if i === 0
+                                      | #{one} (default)&nbsp;
+                                    else
+                                      | | #{one}&nbsp;
+                            else if arg.type_rule
+                              div.col-4.lowercase
+                                | --#{arg.label}
+                                span(style="color: red;") &nbsp;*&nbsp;
+                                | (required):
+                              div.col-8.lowercase
+                                | #{arg.type_rule}&nbsp;
+                                if arg.example
+                                  | (Example: "#{arg.example}")
+                            else
+                              -defaultArgs.push({[argKey]: arg})
+                        if defaultArgs.length > 0
+                          p.muted Default static args:
+                          ul
+                            each argObj in defaultArgs
+                              each arg, argKey in argObj
+                                if arg.type_one_of && arg.type_one_of.length === 1
+                                  li.muted #{argKey}: #{arg.type_one_of[0]}
+                                else if arg.type_string
+                                  li.muted #{argKey}: #{arg.type_string}
```

Comment:
This is unreadable. Four times nested loops and whole bunch of ifs. 
Are there modules in pug templates?

Reply:
yea, i feel you man. Several times i was thinking to re-make it using react/vue/svelte but the change on existing was easier so i left as is
I never planned to support it further, with a hope to build a UI using proper tools for that

Reply:
Tried to simplify by moving into separate sub-templates. hopefully it helps

Reply:
Still hard to read, but at least possible, thanks! :)